crates/bin/node/src/routes/mint.rs

```diff
@@ -76,6 +78,18 @@ impl GrpcState {
         let (expected_amount, state) =
             db_node::mint_quote::get_amount_and_state(&mut tx, quote).await?;
 
+        let quote_response = db_node::mint_quote::build_response_from_db(&mut tx, quote).await?;
+        let expiry = quote_response.expiry;
+
+        let current_time = std::time::SystemTime::now()
+            .duration_since(std::time::UNIX_EPOCH)
+            .map_err(|_| Error::Db(db_node::Error::DbToRuntimeConversion))?
```

Comment:
you can unwrap this one instead. There is no way `std::time::SystemTime::now()` will ever be before `std::time::UNIX_EPOCH`. See: https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.duration_since

Reply:
and we have a utils funtions that does just that in `crates/bin/node/src/utils.rs`